Makefile

```diff
@@ -20,14 +27,19 @@ check-formatting: generate deps vet lint
 
 # Build manager binary
 manager: generate fmt vet
-	go build -o bin/manager github.com/kudobuilder/kudo/cmd/manager
+	go build -ldflags "-X ${GIT_VERSION_PATH}=${GIT_VERSION} \
+             	-X ${GIT_COMMIT_PATH}=${GIT_COMMIT} \
+             	-X ${BUILD_DATE_PATH}=${BUILD_DATE}" \
+             	-o bin/manager github.com/kudobuilder/kudo/cmd/manager
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run: generate fmt vet
-	go run ./cmd/manager/main.go
+	go run -ldflags "-X ${GIT_VERSION_PATH}=${GIT_VERSION} \
+                     -X ${GIT_COMMIT_PATH}=${GIT_COMMIT} \
+                     -X ${BUILD_DATE_PATH}=${BUILD_DATE}" ./cmd/manager/main.go
 
 # Install CRDs into a cluster
-install: manifests
+install: manifests cli
```

Comment:
Not sure I like this one.  This install creates the CRDs on the cluster.  We don't build the `manager` binary when installing the CRDs, that's handled separately.

Reply:
Agreed. We should actually look at goreleaser to build CLI binaries for darwin, linux, and windows:
https://goreleaser.com/ I've used it in the past and it's a great tool for this purpose.

Reply:
Unfortunately, goreleaser won't do our Docker builds though, so we should decide if that's the tool we want to use even though it'll result in 2 different release methods (Makefile for Docker builds, goreleaser for binaries). I think it's ok since goreleaser will give us semantic Github releases but we should talk about it.

Reply:
`make install` is just misleading. What is it actually installing other than CRDs. From a UX perspective I would assume it would install KUDO and all tooling around. The name choice `install` seems to be more misleading in this case. I rather have `bootstrap` or `install-crd` (suggested by @gerred ) for that action. For now I suggest to leave `make install` untouched then and stay with `make cli`.